app/controllers/api/v1/campaigns_controller.rb

```diff
@@ -2,7 +2,9 @@ class Api::V1::CampaignsController < ApplicationController
 
   def signs_recent
     campaign = Campaign.find(params[:id])
-    signs = campaign.signs.present.recent.take(8).map{ |x| { signer_name: x.user_name, body: x.body, signed_at: x.created_at, sequence_no: campaign.signs.where('id <= ?', x).count } }
+    max_count = 8
+    max_count = [params[:max_count].to_i, 32].min if params[:max_count].present?
```

Comment:
```suggestion
    max_count = [params[:max_count]&.to_i, 32].min
```

돌려 보지는 않았는데요. 이게 좀더 의도가 잘 드러나는 듯합니다만.

Reply:
max_count 가 0인 경우도 있을 수 있어서 max_count 파라미터가 없으면 8개를 가져오려고 했습니다. 파라미터로 max_count 가 없으면 결과도 없어야겠군요.